modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/content/processor/test/DefaultExportImportContentProcessorTest.java

```diff
@@ -1150,6 +1150,19 @@ private String _replaceTimestampParameters(String content) {
 					new String[] {StringPool.BLANK, "?" + parameters4}));
 		}
 
+		return _sanitizeURLS(StringUtil.merge(outURLs, StringPool.NEW_LINE));
+	}
+
+	private String _sanitizeURLS(String content) {
+		List<String> urls = ListUtil.fromArray(StringUtil.splitLines(content));
+		List<String> outURLs = new ArrayList<>();
+
+		for (String url : urls) {
+			if (Validator.isNotNull(url) && !url.startsWith(StringPool.POUND)) {
+				outURLs.add(url);
+			}
+		}
+
```

Comment:
I'd call this `extractContentURLs` instead. You're not sanitizing the content, but getting out chunks of it.

Also, can you rename `urls` and `outURLs` to `lines` and `urls`? That would make it clearer the purpose of those lists.

Reply:
Also, it is a bit weird that you're processing all lines, joining them, and the filtering some of them out when you could process only those that you're interested in. Why not filter out those lines that start with `#` in the loop body and get rid of the sanitizeURLS method?

Reply:
Finally, you to prepend lines with `#`; the lines you want to ignore are only the ones that end with`:`, so you could test for that instead.

Reply:
> Also, it is a bit weird that you're processing all lines, joining them, and the filtering some of them out when you could process only those that you're interested in. Why not filter out those lines that start with # in the loop body and get rid of the sanitizeURLS method?


The sanitizeURLS method is needed because it do not enter if the file do not have the `[$TIMESTAMP`, so on the cases of the friendlyURL never will be treated as urls and the "#" 

AND if the line do not have [$TIMESTAMP is simply... ignored!!! (I did not understand this...) so when we merge both files, the friendlyURLs urls will be... deleted... I do not understand this behavior on the tests, but was too afraid to change it

Reply:
> I'd call this extractContentURLs instead. You're not sanitizing the content, but getting out chunks of it.
> 
> Also, can you rename urls and outURLs to lines and urls? That would make it clearer the purpose of those lists.
Ok with lines, but with outURLs I used this name because it's the name that are used all over the test file for the processed urls. But if you think it's better, I'll change it

Reply:
> Finally, you to prepend lines with #; the lines you want to ignore are only the ones that end with:, so you could test for that instead.

they are comments, I like them more that they are well formed :D